#goal definition
---
#result definition
---
#feedback definition
int8 home_count
